### ISSUE TITLE
kajiki include/import DOCTYPE tag

### ISSUE THREAD

**Issue by hemna** [TurboGears/tg2#113]
I have a kajiki (main.xhtml) template that tries to do an include of another xhtml file to keep my 
main template clean and easier to read.  The problem is, that doing the py:include also automatically
prepends all content from that included xhtml file with a bogus DOCTYPE tag.

Here is a snippet of my main template
Followed by the resulting snippet of output.  Note the many DOCTYPE in the resulting output.

```
      <!-- RIGHT BLOCK BEGIN //-->
      <td id="rightblock">
          <table width="100%" border="0" cellspacing="0" cellpadding="0">
              <tr>
                  <td style="vertical-align:top;width:510px">
                      <div id="idMainContent">
                          <py:include href="daily_wx_graph.xhtml" />
                      </div>
                  </td>
                  <td style="vertical-align:top;padding-left:8px;">
                      <py:include href="webcam.xhtml" />
                      <py:include href="satellite.xhtml" />
                      <py:include href="va_radar.xhtml" />
                      <py:include href="ca_radar.xhtml" />
                  </td>
              </tr>
          </table>
      </td>
      <!-- RIGHT BLOCK END //-->
```

My included xhtml files just have html in them.  What ends up being the final result html is

```
      <!--  RIGHT BLOCK END// -->
      <td id="rightblock">
          <table border="0" cellpadding="0" cellspacing="0" width="100%">
              <tr>
                  <td style="vertical-align:top;width:510px">
                      <div id="idMainContent">
                          <!DOCTYPE html>
<div id="idDailyWXGraph">
<script type="text/javascript">
  var data = [{date: "2019-09-27", highTemperature:90.7, lowTemperature: 61.9, rainfall: 0.06, highWindGust: 6.90, highWindGustBearing: 0.00},
  {date: "2019-10-14", highTemperature:78.6, lowTemperature: 0.0, rainfall: 0.00, highWindGust: 5.40, highWindGustBearing: 180.00},
  {date: "2019-10-15", highTemperature:80.6, lowTemperature: 0.0, rainfall: 0.00, highWindGust: 5.40, highWindGustBearing: 0.00},
  {date: "2019-10-16", highTemperature:66.0, lowTemperature: 50.4, rainfall: 0.00, highWindGust: 13.60, highWindGustBearing: 158.00}
  ];

  var graph = new DailyWeatherGraph({data: data,
      container: document.getElementById('idDailyWXGraph'),
      width: 510,
      height: 440,
      temperatureUnit: 'F',
      rainfallUnit: 'inches',
      windUnit: 'mph'});
</script>
</div>
                      </div>
                  </td>
                  <td style="vertical-align:top;padding-left:8px;">
                      <!DOCTYPE html>
<div>
    webcam
<br>
</div>
                      <!DOCTYPE html>
<div>
    Satellite
<br>
</div>
                      <!DOCTYPE html>
<div>
    VA Radar
<br>
</div>
                      <!DOCTYPE html>
<div>
    CA Radar
</div>
                  </td>
              </tr>
          </table>
      </td>
      <!--  RIGHT BLOCK END// -->
```

**Comment by amol-** [comment]
Fixed in https://github.com/nandoflorestan/kajiki/commit/1a1d5d7562d5d9c9c2601825d38f1da7fd4eb5a4

Preparing a new Kajiki release

**Comment by amol-** [comment]
Released Kajiki 0.8.1 with fix for this: https://github.com/nandoflorestan/kajiki/releases/tag/v0.8.1
